packages/opentelemetry-tracing/src/BasicTracerProvider.ts

```diff
@@ -78,7 +83,9 @@ export class BasicTracerProvider implements api.TracerProvider {
   register(config: SDKRegistrationConfig = {}) {
     api.trace.setGlobalTracerProvider(this);
     if (config.propagator === undefined) {
-      config.propagator = new HttpTraceContext();
+      config.propagator = new CompositePropagator({
+        propagators: [new HttpCorrelationContext(), new HttpTraceContext()],
```

Comment:
I think `HttpTraceContext` should be first and then `HttpCorrelationContext`. @dyladan WDYT?

Reply:
It only matters if they may interfere with each other. If anything, I would say the most important (trace context) should be last so if they do interfere it will "win"

Reply:
Ok makes sense.